tests/unit/spec/externalpages/pocket/mobile-nav.es6.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import init from '../../../../../media/js/externalpages/pocket/mobile-nav.es6';
+
+describe('mobile-nav.js', () => {
+    const mobileNav = `<div class="mobile-nav-wrapper" tabindex="-1">
```

Comment:
Simplified nav mocked up here to keep the essential structure and selectors, will need updating if DOM structure changes but any individual link changes are irrelevant